bot/exts/tag_commands.py

```diff
@@ -0,0 +1,51 @@
+import typing
+
+import aiosqlite
+import disnake
+from disnake.ext.commands import Bot, Cog, slash_command
+
+TAGS = typing.Literal[
+    "algos-and-data-structs",
+    "async-and-concurrency",
+    "c-extensions",
+    "cybersecurity",
+    "databases",
+    "data-science-and-ai",
+    "discord-bots",
+    "editors-ides",
+    "esoteric-python",
+    "game-development",
+    "media-processing",
+    "microcontrollers",
+    "networks",
+    "packaging-and-distribution",
+    "software-architecture",
+    "web-development",
+    "tools-and-devops",
+    "type-hinting",
+    "unit-testing",
+    "unix",
+    "user-interfaces",
+]
+
+
+class Tags(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @slash_command(name="add_tag", description="Add a user tag to yourself")
+    async def add_tag(self, interaction: disnake.AppCommandInteraction, tag: TAGS) -> None:
+        await interaction.response.send_message(f"Added tag `{tag}` to `{interaction.user.id}`")
+        tag = "!" + tag
+        conn = await aiosqlite.connect("tags.db")
```

Comment:
We should have a global connection to the database (see `bot.py`)

Reply:
someone else do this bit I'm not sure how this will work

Reply:
assigned myself to the issue